INSTALL.md

```diff
@@ -146,6 +293,38 @@ If running on OS X you should make sure that the Xcode and the Xcode command lin
 
 If you are using Postgres.app on OSX, a database user will be created with your system username. If you want to use this user account, change the `USER` and `ADMIN_USER` settings to your username under the `[postgres]` section of your Qiita config file.
 
+### conda
```

Comment:
I think the postgres section existed before I edited it.

IMO the conda section is generic enough to stay here, because Qiita requires `libgfortran` so it's directly relevant. Plus, most OSX users without a preconfigured system will run into this issue because libgfortran is most likely not in the mainstream conda channels.